### PR TITLE
Issue 1 see breed list

### PR DESCRIPTION
**Pull request by katiefugate**
Screen Shot 2021-04-13 at 3 48 21 PM
Screen Shot 2021-04-13 at 4 00 50 PM

### CODE REVIEW

**Comment by katiefugate** [comment]
Screen Shot 2021-04-13 at 4 33 12 PM
Screen Shot 2021-04-13 at 4 33 28 PM

File: css/layout.css [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  position: absolute;
+  right: 1rem;
+  top: 1.5rem;
+}
+
+.breeds-link {
+  position: absolute;
+  right: 7rem;
+  top: 1.5rem;
+}
```
Comment (codingcodymiller) [inline css/layout.css]: Why the absolute positioning here? What is this for?

File: css/layout.css [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    right: 5rem;
+  }
+
+  .breeds-link {
+    right: 12.5rem;
+  }
+
+  .link-star {
+    right: 8.75rem;
+  }
```
Comment (codingcodymiller) [inline css/layout.css]: 😱 😱 😱 😱

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by katiefugate** [comment]
Screen Shot 2021-04-13 at 5 08 49 PM
Screen Shot 2021-04-13 at 5 10 19 PM

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**